test/integration/test_display_callback.py

```diff
@@ -167,11 +175,11 @@ def test_callback_plugin_no_log_filters(executor, playbook):
     - shell: echo "PUBLIC"
     - shell: echo "PRIVATE"
       no_log: true
-    - uri: url=https://example.org username="PUBLIC" password="PRIVATE"
+    - uri: url=https://example.org user="PUBLIC" password="PRIVATE"
     - copy: content="PRIVATE" dest="/tmp/tmp_no_log"
 '''},  # noqa
 ])
-def test_callback_plugin_task_args_leak(executor, playbook):
+def test_callback_plugin_task_args_leak(executor, playbook, default_envvars):
```

Comment:
I don't think the fixture `default_envvars` is needed for this test? Might be good to clean that up.

Reply:
Yeah, still working out the ins and outs of `pytest`, so I think this was something I tried while trying to get things working.  It works locally for me without it, so I will push up that change.